Source/Editor/Foundation/SceneViewTab/SceneScreenshot.cpp

```diff
@@ -140,14 +147,14 @@ void SceneScreenshot::TakeScreenshotNow(Scene* scene, Camera* camera, const IntV
     });
 }
 
-ea::string SceneScreenshot::GenerateFileName() const
+ea::string SceneScreenshot::GenerateFileName(const IntVector2& size) const
 {
     auto project = owner_->GetProject();
     auto fileSystem = GetSubsystem<FileSystem>();
 
     const ea::string& outputPath = project->GetArtifactsPath();
-    const ea::string screenshotTime = Time::GetTimeStamp("%Y-%m-%d_%H-%M-%S");
-    const ea::string fileNamePrefix = Format("{}Screenshots/{}", outputPath, screenshotTime);
+    const ea::string screenshotTime = Time::GetTimeStamp("%Y-%m-%d/%H-%M-%S");
```

Comment:
If you want to have per-date folder -- fine, but let's keep the YMD in file name as well.